c10/cuda/CUDACachingAllocator.h

```diff
@@ -364,7 +365,7 @@ inline void setMemoryFraction(double fraction, c10::DeviceIndex device) {
   return get()->setMemoryFraction(fraction, device);
 }
 
-inline void emptyCache() {
+inline void emptyCache(MempoolId_t mempool_id = {0, 0}) {
   return get()->emptyCache();
```

Comment:
@ngimel May I know if `mempool_id` should be passed to `get()->emptyCache()` since I see `emptyCache` is used in https://github.com/pytorch/pytorch/blob/93012d2290ed3bf25ac3d17b348dea8f7cb65150/c10/cuda/CUDACachingAllocator.cpp#L4137
If yes, I am glad to fix it in my stack PR https://github.com/pytorch/pytorch/pull/138222